src/pages/HomePage.jsx

```
@@ -1,530 +1,750 @@
-import React from 'react';
-import { COLORS } from '../config/constants';
+import React, { useRef, useCallback } from 'react';
```

ESLint

[error] 'useRef' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)